lib/octokit/client/organizations.rb

```diff
@@ -823,6 +823,26 @@ def unlock_repository(org, id, repo, options = {})
       def billing_actions(org)
         get "#{Organization.path(org)}/settings/billing/actions"
       end
+
+      # Get organization audit log.
+      #
+      # Gets the audit log for an organization.
+      #
+      # @param org [String, Integer] Organization GitHub login or id for which
+      #   to retrieve the audit log.
+      # @option options [String] :include ('all') Filter by event type.
+      #   `all`, `git` or `web`.
+      # @option options [String] :phrase A search phrase.
+      # @option options [String] :order ('desc') The order of audit log events. To list newest events first, specify desc.
+      # To list oldest events first, specify asc.
+      #
+      # @return [Array<Sawyer::Resource>] List of events
+      # @see https://docs.github.com/en/rest/enterprise-admin/audit-log#get-the-audit-log-for-an-enterprise
```

Comment:
This is calling the "Get the audit log for an organization" API, not the enteprise version

```suggestion
      # @see https://docs.github.com/en/enterprise-cloud@latest/rest/orgs/orgs#get-the-audit-log-for-an-organization
```

Reply:
You rock! Thanks for the suggestion.